persistent/picklecache.py

```diff
@@ -156,18 +229,20 @@ def klass_items(self):
     def incrgc(self, ignored=None):
         """ See IPickleCache.
         """
-        target = self.target_size
+        target = self.cache_size
```

Comment:
`target_size` and `cache_size` are aliases for each other. `cache_size` is the official property documented in the interface. Really, because of the fact that ZODB writes to `cache_size`, what I should probably do is drop the property around `cache_size` and stop using `target_size` altogether.